spec/SPEC.md

```diff
@@ -241,59 +241,104 @@ or
 
-### OmniBOR Identifier Embedding
+#### Input Manifest Identifier Embedding
 
-Each build tool should embed into the output artifact a new line terminated,
-lexically ordered, list of OmniBOR identifiers for each mandatory Artifact
+Each build tool should embed into the output artifact a deterministically
+ordered list of Input Manifest Identifiers for each mandatory Artifact
 Identifier Type in a manner:
 
 1. Appropriate to the type of artifact
 2. Generally agreed upon for that artifact
 
-### OmniBOR Document Construction by a Build Tool
+#### Input Manifest Construction by a Build Tool
 
-A build tool creating an output artifact must compute a OmniBOR Document of
-each mandatory artifact ID type.
+A build tool creating an output artifact must compute an Input Manifest of
+each mandatory artifact identifier type.
 
 For each input artifact the build tool must:
 
-1. Compute the git object id of the input - `${artifact identifier}`
-2. Examine the input for an embedded OmniBOR Document Identifier -
-  `${omnibor document identifier}`
+1. Compute the artifact identifier of the input - `${artifact identifier}`
+2. Examine the input for an embedded Input Manifest Identifier -
+  `${input manifest identifier}`
+
+The build tool must persist an Input Manifest using the
+`${artifact identifier}` and `${input manifest identifier}` for each input.
+
+#### Input Manifest Examples
+
+```gitoid:blob:sha1
+blob 06a6891154fff74e1ddb6245f4a0467b09c617c5
+blob 06dd79bc831bb06a6267a36ad2d62beccd7900b2 bom a9a64def763517df596fbb4348a8561069b5dc4b
+blob 0bc39408c1e5feaadd6f0420d14324b477420b93
+blob 15acd4427ca14000111aad5071563bc7f2dc09f4
+blob 1be90e6fab4ab9b7dd3b27cea5bb1fe29acc0204
+blob 1d8a4e28d1b62a2bfeba837fe18422cd106e6ddf bom 5bda8237d1676df0a2d0b8682d40f99a27ef5b13
+blob 28488e0b05954ccf87c779f5f9258987e4d68ac5
+blob 2c0cde251f1a9f05563a5f7a7f32588f04aaa235
+```
+
+```gitoid:blob:sha256
+blob 09c825ac02df9150e4f93d12ba1da5d1ff5846c3e62503c814aa3a300c535772
+blob 230f3515d1306690815bd9c3da0d15d8b6fcf43894d17100eb44b6d329a92f61
+blob 2f4a51b16b76bbc87c4c27af8ae062b1b50b280f1ab78e3eec155334588dc88e bom 4f3a822f776412c049dda53c3277bf2225b51b805ce8a99222af23a7d9f55636
+blob c71d239df91726fc519c6eb72d318ec65820627232b2f796219e87dcf35d0ab4
+blob f47ffb3518f236eea6525fd29f057ddd5cda1bb803ccc662e6bc5925afd1e4af
+```
+
+### Artifact Dependency Graph (ADG)
+
+The Artifact Dependency Graph (ADG) of an artifact is the recursive DAG
+(Directed Acyclic Graph) of all the "input artifacts" that are transformed
+by a build tool into that artifact. It includes the direct input artifacts,
+and the recursive set of input artifacts to each input artifact, all the way
+down the graph.  
+
+Concretely the Artifact Dependency Graph (ADG) of an Artifact is:
+
+- The set of Input Manifests defined by:
+  -  The Input Manifest of the Artifact
+  -  Any Input Manifest referenced in an Input Manifest in the set
```

Comment:
```suggestion
- Any Input Manifest referenced in an Input Manifest in the set (ie the transitive closure of the Input Manifests)
```

Reply:
Fixed in https://github.com/omnibor/spec/pull/51/commits/28f03669fa6af228932e4bdd80fba5158e92ed36